.github/workflows/build-bundle.yml

```diff
@@ -0,0 +1,39 @@
+name: Build a static wallet bundle
+
+on:
+  push:
+    paths:
+      - 'src/**'
```

Comment:
@sideninja I think we should further limit this action to only run on commits to `master`.

For context, @10thfloor and I added this filter for `src/**` so that the zip bundle would only be created when the actual app changes.

@10thfloor Can you open a PR (into this branch) that moves all the Next.js files into the `src` directory?

Further limiting the bundle to master means that it won't create a bunch of PRs on working branches.

Reply:
oh yeah this should be limited to new version releases (it was) I guess it was slipped in during testing. My bad will fix

Reply:
I actually removed the tag trigger on purpose.

My reasoning was that I think the bundle should actually be built and committed to master _before_ we tag the repository. This is because the Go module is versioned on tag, and therefor Go module e.g. v1.0 won't actually have the v1.0 zip bundle.

If it triggers on pushes to master, then it will build a new bundle on each PR and we can tag the ones that are tested and working, creating a complete release of the Go package.

Reply:
Ohh, nice catch didn't think of that. It should be pre-release action yeah, a release draft would also work, but I guess it's better to just target changes in that dir on the master branch so we don't forget to include it.

Reply:
I can make this work @psiemens if you wish, but I believe we will have to use another GHA because changing multiple events with AND condition is not possible as far as I know, so we should make it be triggered on master changes and then validate the folder of our interest changed by using action like https://github.com/tj-actions/changed-files

Reply:
I think it should be possible to have two filters according to the [workflow docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore). It doesn't seem to be very explicit behaviour though

> If you define both branches/branches-ignore and paths, the workflow will only run when both filters are satisfied.

```
on:
  push:
    branches:
      - 'master'
    paths:
      - 'src/**'

Reply:
Ohh right, I was reading the section for `on` for multiple events https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-multiple-events 
that's good then. I will edit it in, thank you.